google/resumable_media/common.py

```diff
@@ -142,6 +146,8 @@ def __init__(
         self.max_sleep = max_sleep
         self.max_cumulative_retry = max_cumulative_retry
         self.max_retries = max_retries
+        self.initial_delay = initial_delay
+        self.multiplier = multiplier
```

Comment:
This bit needs explicit tests in `tests/unit/test_common` for the ctor defaults and for explicitly-passed values.